### ISSUE TITLE
Anchor links in search results - can we and should we?

### ISSUE THREAD

**Issue by thepegisin** [cagov/covid19#5846]

**Comment by aaronhans** [comment]
The slack conversation on this seemed like we were talking about a quick answer. This is pretty easy to handle there. Were you interested in this being part of the main search results section instead of quick answers @thepegisin? That is more complex

**Comment by thepegisin** [comment]
Yes, instead of quick answers. I know we have elevated quick answers in search to appear first, and I understand why, but that's actually problematic. In content design, we put the most important and most-needed content in body text, and put content that affects fewer people in FAQs. But then we elevate those FAQs to the top in search. So if someone searches on "quarantine," the best answer for how to quarantine doesn't appear first (the Testing page), it appears after FAQs that describe finer details like what paid leave you can expect.

Even if the Testing page did appear first in a search on "quarantine," it's likely not obvious that this is a good answer. So it would be great if a search on "quarantine" brought up an anchor link named "When to isolate or quarantine."

**Comment by aaronhans** [comment]
Great points about quick answers getting in the way @thepegisin. I believe @jaredwyee has the search initiative on his plate. We may need to switch search engine providers to fix this(We have access to several via Azure/AWS). There is also a CDT initiative to improve search @rkojik is heading, we could see if he has new tools for us. cc @jbum

**Comment by jaredwyee** [comment]
@aaronhans I'm not working on search

**Comment by aaronhans** [comment]
I checked in with CDT search folks and they didn't have any quick fixes. Tagging @mediajunkie on this because the point that current implementation of quick answers is not optimal is an important. Will let product team drive this so we can get  feedback to determine optimal engineering approach.

**Comment by aaronhans** [comment]
Would it be better to move quick answers below the main search results?

**Comment by aaronhans** [comment]
We could create two separate quick answers datasets, one which is more general and then one we put at the bottom which is more specific FAQs. We can also test different main content search tools

**Comment by thepegisin** [comment]
We could try putting quick answers under main search results, but they should still appear on the first page, IMHO. Maybe after the first 5 results? Or the first 2?

**Comment by mediajunkie** [comment]
I'm happy to try either! 

First, is there something we can benchmark to capture the problem (user feedback where it's clear the Q/A distracted them from the main point, etc.), so when we try changing it, after a while we can check or measure it again to see if the problem got better?

**Comment by rkojik** [comment]
The creative (maybe not-so creative... maybe even crazy...) way I would've done this if I had pages... created a blank one with a server-side-include redirect to the page/anchor. Though, this would require that SSI redirect page be linked to _somewhere_ in content so that Google crawls it.
